index.js

```diff
@@ -19,17 +19,37 @@ module.exports.env = function() {
 };
 
 module.exports.os = function() {
-  if (typeof os.userInfo === 'function') {
-    return os.userInfo().username;
+  try {
+    if (typeof os.userInfo === 'function') {
+      return os.userInfo().username;
+    }
+  } catch(e) {
+    if (e !== null && typeof e === 'object' && e.code === 'ENOENT') {
+      console.log("UserInfo does not exist");
```

Comment:
I would prefer to not provide logging by default for non-actionable things.

Ether:

a) remove the log
b) use [node-debug](https://www.npmjs.com/package/debug) to do the logging, which makes it "opt in" <-- I like this option the best